Orbital-Docs/_pages/docs/quick-start-guide/installing.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: single
+title: Installation
+permalink: /docs/installation/
+sidebar:
+  nav: 'docs'
+---
+
+**Note: Orbital uses an in-memory datastore. Please save or export all data (via Download Mockdefinitions or Export Mock to Server) before exiting, otherwise it will be lost.**
+
+It's very easy to get started; just [download the Docker Engine for Linux, Mac, and Windows](https://hub.docker.com/?overlay=onboarding):
+
+## Using Docker
+
+`docker run -p 4200:80 focisolutions/orbitaldesigner`
+
+`docker run -p 5000:80 focisolutions/orbitalmock`
+
+The mock definition designer will be running on `http://localhost:4200`, and the server on `http://localhost:5000`.
+
+## Using Docker Compose
+
+Install [Docker Compose](https://docs.docker.com/compose/install/)
+
+`git clone https://github.com/FociSolutions/Orbital.git`
+
+`cd Orbital`
+
+`docker-compose build`
+
+`docker-compose up -d`
+
+The mock definition designer will be running on `http://localhost:4200`, and the server on `http://localhost:5001`.
```

Comment:
we need to either change the server url to `http://localhost:5000` or change the previous line 17 to `docker run -p 5001:80 focisolutions/orbitalmock`. Although, can we verify which is the correct one? I remember having issue with running the docker image in port 5000.

Reply:
Thank you! Will change.

Reply:
Changed. I didn't have any issues running on port 5000 (I've always used the `README` instructions when running the containers locally.)